modules/sdk-coin-eth/src/eth.ts

```diff
@@ -220,6 +235,8 @@ export interface RecoverOptions {
   eip1559?: EIP1559;
   replayProtectionOptions?: ReplayProtectionOptions;
   isTss?: boolean;
+  feeAddress?: string;
```

Comment:
`feeAddress` cannot be optional as it is used in `getAddressNonce` and `queryAddressBalance`

Reply:
May need to extend this with another type or check if added as required does not create a breaking change.

Reply:
Making it directly as a required field in RecoverOptions will break other types of recovery.

As there are different types of recovery happening through this recover method I have defined feeAddress as optional.
Particularly for evm cross chain recovery this field is required and hence I have used validateEvmBasedRecoveryParams() in recoverEthLikeforEvmBasedRecovery() which checks that feeAddress is required.